CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 0.2.1
+* Add Python 3 support
```

Comment:
are we adding python 3 support or replacing 2 with 3?  New code won't work with Python 2, right?

Reply:
it will work with both py2 and py3